### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
everything looks good to me , good work

File: Home.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+.navbar-list {
+    display: flex;
+    margin: 0.2rem 2.5rem;
+    font-weight: 600;
+}
+ul {
+    list-style-type: none;
+    padding: 1rem 0rem;
+}
+li {
```
Comment (anupkgurung) [inline Home.css:98]: Do not add style directly to element.

File: mycart/cart.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
+:root {
```
Comment (anupkgurung) [inline mycart/cart.css:1]: keep all the variables in one file and use that.

File: mycart/cart.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    --grey-color: rgb(138, 136, 136);
+    --primary-color: rgb(83, 83, 221);
+    --card-background: rgb(244, 51, 151);
+
+    --btn-background-color: rgb(235, 229, 229);
+    --white: rgb(248, 247, 247);
+    --nav-background: rgb(252, 250, 250);
+    --card-buying-background: #f1f1f1;
+}
+* {
```
Comment (anupkgurung) [inline mycart/cart.css:12]: styles for all(*) should also be in there in one and not individually for every file

File: mycart/cart.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    font-family: sans-serif;
+}
+.card-img {
+    height: 14.3rem;
```
Comment (anupkgurung) [inline mycart/cart.css:16]: why height 14.3 ? was it calculated ?

File: mycart/cart.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    gap: 3rem;
+}
+.card-buying-container {
+    background-color: var(--card-buying-background);
+}
+.card-buying-inner-container {
+    margin: 1rem;
+}
+.card-buying-detail {
+    display: flex;
```
Comment (anupkgurung) [inline mycart/cart.css:80]: display flex and flex direction are used widely move it into a common or utility class.

File: mycart/cart.css [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    position: relative;
+    top: 3px;
+}
+.card-component {
+    display: flex;
+
+    box-shadow: var(--main-box-shadow);
+    width: 28rem;
+    height: 14.3rem;
+    border-radius: 4px;
```
Comment (anupkgurung) [inline mycart/cart.css:44]: how is height calculated 14.3?

File: mycart/cart.html [hunk 7]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My cart</title>
```
Comment (anupkgurung) [inline mycart/cart.html:7]: Use meta tags for description also.

File: mycart/cart.html [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+                    <div class="card-quantity">
+                        <span class="quantity">Quantity:</span>
+                        <span class="material-icons md-18">
+                            remove_circle_outline
+                        </span>
+                        <span class="numeric">
+                            1
+                        </span>
+                        <span class="material-icons md-18">
+                            add_circle_outline
```
Comment (anupkgurung) [inline mycart/cart.html:72]: Use proper indentation or do format the code before commit for good readability.

File: Home.html [hunk 9]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (anupkgurung) [inline Home.html:6]: use meta for description also

File: Home.css [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+    font-size: 2rem;
+    font-weight: 600;
+
+    text-align: center;
+    margin: -1.1rem 0.5rem;
+}
+.home-card {
+    display: flex;
+    margin: 1rem 13rem;
+
```
Comment (anupkgurung) [inline Home.css:176]: remove unused space.

File: Home.css [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+    width: 17%;
+    justify-content: space-between;
+}
+.input-inner-container {
+    padding-right: 1rem;
+    max-width: 25rem;
+    height: 3.2rem;
+    text-align: center;
+    padding: auto;
+    line-height: 1.5;
```
Comment (anupkgurung) [inline Home.css:44]: why padding auto and padding-right are used simultaneously ?

**Review by anupkgurung — CHANGES_REQUESTED** [review]
Please go through the comments

File: Home.css (hunk 1 above)
Comment (ishant005) [reply Home.css:98]: Ok .will refactor it

File: mycart/cart.css (hunk 2 above)
Comment (ishant005) [reply mycart/cart.css:1]: ok

File: mycart/cart.css (hunk 4 above)
Comment (ishant005) [reply mycart/cart.css:16]: resolved

**Pull request merged**